skrifa/src/patchmap.rs

```diff
@@ -278,8 +419,8 @@ impl PatchEncoding {
 /// the numeric index is supported.
 #[derive(Debug, Clone, Eq, PartialEq, Hash)]
 pub struct PatchUri {
-    template: String,
-    index: u32,
+    template: String, // TODO: Make this a reference?
```

Comment:
this is definitely doable; whether it's worth doing really depends on how hot the codepath is where this gets used. In any case it would require threading the lifetime of the font data through all the intermediate calls and types.

Reply:
Sounds good, I figured I'd hold off on this until I have the other side (the client which uses this code) and then see if it makes sense from there to enable this. There could be hundreds to thousands of entries in a IFT font so not duplicating this template on each is probably fairly valuable.